packages/office-ui-fabric-react/src/components/Persona/Persona.base.tsx

```diff
@@ -119,14 +130,17 @@ export class PersonaBase extends BaseComponent<IPersonaProps, {}> {
     );
 
     return (
-      <div
-        { ...divProps }
-        className={ classNames.root }
-        style={ coinSize ? { height: coinSize, minWidth: coinSize } : undefined }
-      >
-        <PersonaCoin { ...personaCoinProps } />
-        { (!hidePersonaDetails || (size === PersonaSize.size10 || size === PersonaSize.tiny)) && personaDetails }
-      </div>
+      <FocusZone componentRef={ this._focusZone } >
```

Comment:
Could you possibly explain how we enable focus on components like this if I shouldn't use a FocusZone? I thought that was the point of a FocusZone.

Reply:
You don't add focus to a presentational component. You wrap it in a button or something which is actionable. Presentation components are not interactable.

Reply:
@jordandrako  The purpose of `FocusZone` is:

FocusZones abstract arrow key navigation behaviors. Tabbable elements (buttons, anchors, and elements with data-is-focusable='true' attributes) are considered when pressing directional arrow keys and focus is moved appropriately. Tabbing to a zone sets focus only to the current "active" element, making it simple to use the tab key to transition from one zone to the next, rather than through every focusable element.